content/release-notes/2023-06-27-storage-and-compute.md

```diff
@@ -0,0 +1,12 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Added support for the [pg_uuidv7](https://github.com/fboulnois/pg_uuidv7) extension, which enables creating [version 7 UUIDs](https://www.ietf.org/archive/id/draft-ietf-uuidrev-rfc4122bis-00.html#name-uuid-version-7) in PostgreSQL.
+- Compute: Added support for the [pg_roaringbitmap](https://github.com/ChenHuajun/pg_roaringbitmap) extension. Roaring roaring bitmaps are a type of compressed bitmap that generally surpass traditional compressed bitmaps.
+
+### Bug fixes
+
+- Compute: Resolved a problem where roles created in the Neon console were not granted database privileges until the subsequent compute restart.
```

Comment:
it wasn't fixed yet. Let's just skip this line

Reply:
Thank you. Removed.